migrate-core/src/lib.rs

```diff
@@ -1,13 +1,13 @@
-//! The core crate that exposes APIs designed for full programmatic
+//! Core crate that exposes APIs designed for full programmatic
 //! configuration of migrations.
 //! See [`migrate`] crate if you need a CLI wrapper instead.
 //!
-//! The crate is built on top of [`migrate_state`] and the ideas outlined there.
+//! Crate is built on top of [`migrate_state`] and the ideas outlined there.
```

Comment:
```suggestion
//! The crate is built on top of [`migrate_state`] and the ideas outlined there.
```
`The` here is important to denote that it's the crate where this documentation is written in. I don't remember where, but I think I've heard the rule "if you can replace `the` with `this`, then it's correct"

Reply:
I guess you are right